modmesh/app/euler1d.py

```diff
@@ -442,288 +453,266 @@ def _update_single_figure(self):
                     ax_new.yaxis.set_major_formatter((FormatStrFormatter
                                                       ('%.2f')))
                 select_num += 1
-            lines.append(data.ana)
-            lines.append(data.num)
 
         ax.set_xlabel("distance (m)")
         ax.grid()
-        canvas.installEventFilter(self)
 
         # These parameters are the results obtained by my tuning on GUI
         fig.subplots_adjust(left=0.1,
                             right=0.97 - (self.checkbox_select_num - 1) * 0.1,
                             bottom=0.093, top=0.976,
                             wspace=0.2, hspace=0.2)
 
-        prev_fig = self.figure_layout.takeAt(0)
-
-        if prev_fig:
-            prev_fig.widget().deleteLater()
-
         self.update_lines()
-        self.figure_layout.addWidget(canvas)
-
-        return lines
 
-    def _build_checkbox(self, layout):
-        layout.setSpacing(50)
-        for name, data_line in self.data_lines.items():
-            check_box = QCheckBox(name)
-            if data_line[1]:
-                check_box.toggle()
-            check_box.clicked.connect(self._checkbox_cb)
-            layout.addWidget(check_box)
```

Comment:
Data line visualization selection will be moved to the configuration window of the plotting area later.
Therefore the checkbox is removed.